frontend/apps/web/app/(mgmt)/[account]/connections/components/ConnectionsTable/columns.tsx

```diff
@@ -67,7 +67,9 @@ export function getColumns(
         return (
           <div className="flex space-x-2">
             <span className="max-w-[500px] truncate font-medium">
-              {getCategory(row.original.connectionConfig)}
+              {getCategory(row.original.connectionConfig) == 'Mysql'
+                ? 'MySQL'
+                : getCategory(row.original.connectionConfig)}
```

Comment:
why not just have the `getCategory` function return `MySQL`? That is the entire point of that method to return the value to be displayed in the table :D

Reply:
ah i missed that it was a func we wrote, thought it was from tanstack, that's much easier!